### ISSUE TITLE
amp-validator: Should non-Hardware Accelerated CSS be valid?

### ISSUE THREAD

**Issue by torch2424** [ampproject/amphtml#22061]
This was brought up to me during AMP Conf by a partner. They were hopping to get the AMP validator to ensure only hardware accelerated CSS animations would be allowed by their pages, but found we don't enforce that. Do you think it would be a good idea to do so? 

cc @Gregable @honeybadgerdontcare 

Feel free to close the issue once a decision is made 👍

**Comment by honeybadgerdontcare** [comment]
I believe there was a desire to only allow hardware accelerated animations but it isn't necessarily enforced by the validator. @dvoytenko may know more about the initial vision/restrictions.

**Comment by aghassemi** [comment]
We discussed it here recently https://github.com/ampproject/amphtml/issues/21933 .
tl;dr: yes but because initially we were not parsing CSS in validator it was not possible and now it is too late.

**Comment by honeybadgerdontcare** [comment]
@aghassemi thank you for pointing out that other thread. This does seem like something we'd like to do starting from square one, but since we're at where we are, then it's not going to happen.

**Comment by Gregable** [comment]
Drive-by comment: I suppose it could be something we add to new formats (aka amp4email) or `<style amp-keyframes>`, etc. Not reopening though as we have no such thing planned in the future.
